core/src/main/java/apoc/util/DateParseUtil.java

```
@@ -28,7 +28,8 @@ public static TemporalAccessor dateParse(String value, Class<? extends TemporalA
                             return getParse(date, value);
                         }
                     } catch (Exception e) {
-                        continue;
+                        // Ignore here as we are in a loop checking formats
+                        // if all formats fail, a later exception will be called
                     }
```

Same, empty catch block still